doc/spec/draft-lcurley-moq-lite.md

```
@@ -0,0 +1,675 @@
+
+A conflict can occur when a relay tries to serve multiple downstream subscriptions from a single upstream subscription.
+Any upstream subscription SHOULD use the publisher priority, not some combination of different subscriber priorities.
+
+Rather than try to explain everything, here's an example:
+
+**Example:**
+There are two people in a conference call, Ali and Bob.
+
+We subscribe to both of their audio tracks with priority 2 and video tracks with priority 1.
+This will cause equal priority for `Ali` and `Bob` while prioritizing audio.
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)